arreglo/src/main/java/com/globant/model/Payment.java

```diff
@@ -0,0 +1,37 @@
+package com.globant.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "payments")
+public class Payment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+    private float amount;
+
+    public Payment() {
```

Comment:
remove all geters and setters use the @DaTa from lombok and use the @NoArgsConstructor & @AllArgsConstructor aswell